src/nickelmenu.cc

```diff
@@ -206,6 +207,15 @@ extern "C" __attribute__((visibility("default"))) MenuTextItem* _nm_menu_hook(vo
     return AbstractNickelMenuController_createMenuTextItem(_this, menu, label, checkable, checked, thingy);
 }
 
+QString nm_menu_pixmap(const char *path, const char *fallback) {
+    QPixmap a;
+    if (!a.load(QString(path))) {
+        NM_LOG("nm_menu_pixmap: error loading '%s', falling back to '%s': %s", qPrintable(path), qPrintable(fallback), QFile::exists(path) ? "failed to load image" : "image does not exist");
+        return fallback;
```

Comment:
`return QString(fallback)`?

Reply:
Implicit conversion will handle that, but I like to be explicit, so I'll fix that.

Reply:
Cool. I don't know enough about C++ to know what's implicit and what's not, so I tend to err on the side of being explicit,

Reply:
Done.